### PR TITLE
Update allocation add/remove blobber

### PR DESCRIPTION
**Pull request by Sriep**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/1053
- blobber:
- gosdk: https://github.com/0chain/gosdk/pull/410
- system_test:
- zboxcli: https://github.com/0chain/zboxcli/pull/189
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | update-ba  |
| zboxcli  | update-ba  |
| zwalletcli  | staging  |
| 0chain  | update-blobbers  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | update-ba  |
| zboxcli  | update-ba  |
| zwalletcli  | staging  |
| 0chain  | update-blobbers  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: cmd/updateallocation.go [hunk 1]
```diff
@@ -26,6 +26,24 @@ var updateAllocationCmd = &cobra.Command{
 			log.Fatal("invalid 'allocation_id' flag: ", err)
 		}
 
+		var addBlobberId, removeBlobberId string
+		if flags.Changed("add_blobber") == true {
```
Comment (peterlimg) [inline cmd/updateallocation.go]: Could be `if flags.Changed("add_blobber") {`

File: cmd/updateallocation.go [hunk 2]
```diff
@@ -26,6 +26,24 @@ var updateAllocationCmd = &cobra.Command{
...
+			addBlobberId, err = flags.GetString("add_blobber")
+			if err != nil {
+				log.Fatal("invalid 'add_blobber' flag: ", err)
+			}
+			if flags.Changed("remove_blobber") == true {
```
Comment (peterlimg) [inline cmd/updateallocation.go]: Same as "add_blobber"

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**